### PR TITLE
Us146b addition of buid for conveyancer

### PR DESCRIPTION
**Pull request by nigelpain**

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -6,11 +6,21 @@
...
+        'http://0.0.0.0:9090/webseal')
+  end
+
+  def self.deed_api_casework
+    (ENV['DEED_API_CASEWORK_URL'] ||
         'http://0.0.0.0:9020')
   end
 
+  def self.deed_api_buid_b
+    (ENV['DEED_API_BUID_A_URL'] ||
```
Comment (danielmarker) [inline features/support/env.rb]: Is this correct for 2 Env's to have the same name but different URI?

Comment (nigelpain) [inline features/support/env.rb]: hah nice spot! serves me right for rushing things =)

**Comment by flynnbops** [comment]
LGTM :+1:

**Comment by vaakinmade** [comment]
:+1:

**Comment by tsangsimon** [comment]
:+1:

**Pull request merged**